TwitchDownloaderWPF/Translations/Strings.es.resx

```diff
@@ -764,6 +764,6 @@
     <value>Alterna el color de fondo de cada mensaje de chat para ayudar a distinguirlos.</value>
   </data>
   <data name="EncodeClipMetadata" xml:space="preserve">
-    <value>Inclure les métadonnées:</value>
```

Comment:
Looks like I accidentally put the French translation in the Spanish file, whoops.

Reply:
I reviewed it and translated it